examples/cache/service.py

```diff
@@ -0,0 +1,42 @@
+from flask import Flask
+from flask import request
+from flask import make_response, abort
+import yaml
+import os
+import requests
+import socket
+import sys
+import datetime
+
+app = Flask(__name__)
+
+
+@app.route('/service/<service_number>/<response_id>')
+def get(service_number, response_id):
+  stored_response = yaml.load(open('/etc/responses.yaml', 'r')).get(response_id)
+
+  if stored_response is None:
+    abort(404, 'No response found with the given id')
+
+  response = make_response(stored_response.get('body') + '\n')
+  if stored_response.get('headers'):
+    response.headers.update(stored_response.get('headers'))
+
+  # Generate etag header
+  response.add_etag()
+
+  # Append the date of response generation
+  body_with_date = response.get_data(as_text=True) + 'Response body generated at: ' \
```

Comment:
nit: interpolation is usually a better approach with python

Reply:
What do you mean by interpolation here?

Reply:
ie, one of

```
"some %s containing %s" % ("string", "foo")

"some {} containing {}".format("string", "foo")
```

the latter is the officially preferred way